docs-src/docs/intro.md

```diff
@@ -33,4 +33,4 @@ Your running Layer0 applications. We also use the term `service` for tools such
 
 ### Environments
 
-Logical groupings of services. Typically, you would make a single environment for each tier of your application, such as `dev`, `staging`, and `prod`.
+Logical groupings of services. Typically, you would make a single environment for each tier of your application, such as `dev`, `staging`, and `prod`. Additionally an environment can be either static or dynamic. Static environments should be used when you require more fine grained control over an EC2 instance that a container will run on. Dynamic environments (the default) allow you to run containers without having to worry about managing and scaling clusters of EC2 instances.
```

Comment:
This is a good start, but I think we're going to need additional docs to explain this difference --- pros and cons of either approach in detail, etc. That can be a separate issue we create, though.

Reply:
added this issue for now https://github.com/quintilesims/layer0/issues/537